src/server/req_jobobit.c

```diff
@@ -649,7 +650,12 @@ rel_resc(job *pjob)
 
 	/* Mark that scheduler should be called */
 
-	set_scheduler_flag(SCH_SCHEDULE_TERM);
+	if (find_assoc_sched_pj(pjob, &psched))
+		set_scheduler_flag(SCH_SCHEDULE_TERM, psched);
+	else {
+		sprintf(log_buffer, "Unable to reach scheduler associated with job %s", pjob->ji_qs.ji_jobid);
```

Comment:
In this log also we should mention partition instead of jobid

Reply:
Implemented.